tools/qualimap.wdl

```diff
@@ -9,32 +9,36 @@ task bamqc {
     input {
         File bam
         Int ncpu = 1
-        String prefix = basename(bam, ".bam")
+        String prefix = basename(bam, ".bam") + "_qualimap_bamqc_results"
         Int max_retries = 1
         Int memory_gb = 8
         Int? disk_size_gb
     }
-    Int java_heap_size = ceil(memory_gb * 0.9)
 
+    Int java_heap_size = ceil(memory_gb * 0.9)
+    String out_directory = select_first([prefix, "qualimap_bamqc_results"])
```

Comment:
Instead of stripping the components in the multiqc step and writing to a folder, why don't we just use whatever top-level folder named was tar-ed up in this step (and don't even deal with creating a new folder in multiqc). That way, at least we isolate the problem of picking a name to here. We could even just set the prefix as `basename(bam, ".bam")` by default, which accomplishes your solution in the Slack channel